pye3d/eye_model/base.py

```diff
@@ -158,7 +158,7 @@ def estimate_sphere_center_3d(
             squared_residuals = np.einsum(
                 "ikj,ijk->i", np.transpose(deltas, (0, 2, 1)), tmp
             )
-            rms_residual = np.mean(np.sqrt(squared_residuals))
+            rms_residual = np.mean(np.sqrt(np.abs(squared_residuals)))
```

Comment:
Looking at this, I wonder whether we could make it more explicit that this is just to avoid the numerical problem of near-zero values being negative. Could we use np.clip(square_residuals, 0.0) instead?